src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_resolver.py

```diff
@@ -405,6 +405,23 @@ class SetResolver:
         Resolves a set as dict id(object)->object
     '''
 
+    def get_contents_debug_adapter_protocol(self, obj, fmt=None):
+        ret = []
+
+        for i, item in enumerate(obj):
+            ret.append((str(id(item)), item, None))
+
+            if i > MAX_ITEMS_TO_HANDLE:
```

Comment:
This should probably use, `start` and `count` from variables request. We should have a separate task to handle that.

Reply:
Created: https://github.com/Microsoft/ptvsd/issues/1392 to handle that.